workitem/workitemtype.go

```diff
@@ -190,33 +189,6 @@ func (wit WorkItemType) Equal(u convert.Equaler) bool {
 	return wit.SpaceID == other.SpaceID
 }
 
-// ConvertWorkItemStorageToModel converts a workItem from the storage/persistence layer into a workItem of the model domain layer
```

Comment:
Sorry @baijum for the confusion can you revert this change please? I only meant to remove the function on the WorkItemType if it wasn't used. But it was used and so...

Reply:
@kwk There is only one use and that was from the `ConvertWorkItemStorageToModel` function. And as you can see all tests are passing. Do you still want to revert it?

Reply:
Yes please.

Reply:
Reverted. 0716c2883117a100cd9c1fa22fa4729e1ba98c84